articles/quickstarts/install-command-line.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Develop with Q# applications
+title: Develop with Q# applications in an IDE
```

Comment:
The other titles are all "Develop with Q# and ...".  This is consistent but may need tweaking.

Reply:
@geduardo mentioned that the old title, "Develop with Q# command-line applications", probably got changed at some point in the restructuring. Thoughts on going back to that @KittyYeungQ ?